atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -317,6 +322,16 @@ def get_build(self, metadata, worker_metadatas):
                 else:
                     extra['filesystem_koji_task_id'] = task_id
 
+        parent_id = self.get_parent_image_koji_build_id()
+        if parent_id is not None:
+            try:
+                parent_id = int(parent_id)
+            except ValueError:
+                self.log.exception("invalid koji parent id %r", parent_id)
+            else:
+                extra.setdefault('image', {})
+                extra['image']['parent-image-koji-build-id'] = parent_id
```

Comment:
Other metadata keys are in "snake_case" not "kebab-case". I think we should use underscores in the name, not dashes.

Reply:
Still needs underscores, not dashes, here.